app/vep/models/pipeline_model.py

```diff
@@ -91,7 +93,7 @@ def create_config_ini_file(self, dir):
 class PipelineStatus(BaseModel):
     submission_id: str
     status: str = Field(
-        alias=AliasPath("status", "workflow", "status"), default="FAILED"
```

Comment:
`AliasPath` can only be used with `validation_alias` (`alias` expects a string)

Reply:
I don't think this is the case. We use aliaspath the same way on metadata and it's working.

Reply:
Agree with @Mehrnaz-Charkhchi  alias, aliaspath is just the mechanism to extract the value.

Reply:
Indeed `AliasPath` can be used to extract information, but not to serialize it (according to documentation and MyPy reports), so the correct way is to use `validation_alias` instead of `alias` param. Same with metadata.